subsys/lorawan/lorawan.c

```diff
@@ -0,0 +1,434 @@
+/*
+ * Copyright (c) 2019 Manivannan Sadhasivam <mani@kernel.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <errno.h>
+#include <lorawan.h>
+#include <zephyr.h>
+
+#include <LoRaMac.h>
+
+#ifdef CONFIG_LORAMAC_REGION_AS923
+	#define CONFIG_LORAWAN_REGION LORAMAC_REGION_AS923
```

Comment:
It's not OK to define macro's named CONFIG_*.

CONFIG_ is a reserved namespace for Kconfig.

Reply:
Okay